src/streaming/models/MediaPlayerModel.js

```diff
@@ -115,23 +113,16 @@ function MediaPlayerModel() {
         }
     }
 
-    function getStableBufferTime() {
```

Comment:
it means that stableBufferTime will be the same if low latency is enabled or fast switch...

Reply:
You are right.